.github/ISSUE_TEMPLATE/Currency_update_request.yml

```diff
@@ -0,0 +1,35 @@
+name: Currency Update Request
+title: "[Currency_Update]: "
+description: "Update request to add support for specific version on ppc64le for available currency package Or fix currency "
```

Comment:
How about just:  "Request to add, fix, or update a package in the Currency Portal"?

Reply:
ok, we can do that.
As we have separate template for add , Iam making it like `Request to fix, or update a package in the Currency Portal`